app/views/docs/functions-develop.phtml

```diff
@@ -444,10 +481,10 @@ public class Handler {
 </p>
 
 <p>You'll find these properties in the context object.</p>
-<table class="full text-size-small">
+<table cellspacing="0" cellpadding="0" border="0"  class="full text-size-small">
     <thead>
         <tr>
-            <td>Property</td>
+            <td style="width: 250px">Property</td>
```

Comment:
somehow, the error() row got an extra column added to it. Would you please fix that?

Reply:
removed 👌